keybase/config.go

```diff
@@ -48,6 +48,8 @@ type store struct {
 	Auto bool `json:"auto"`
 	// AutoSet is whether a user set the Auto config
 	AutoSet bool `json:"autoSet"`
+	// AutoOverride is whether the current auto setting should be temporarily overridden
+	AutoOverride bool `json:"autoOverride"`
```

Comment:
Should we move this out of the store and into `config` struct?

We never want to store this right?